docs/source/lfp.rst

```diff
@@ -1,16 +1,18 @@
-:orphan:
+=======================================================
```

Comment:
ah there was already an lfp file. I did not find it (probably for the online part lol)

Reply:
(you can close this once read)